app/assets/stylesheets/partials/_pagination.scss

```diff
@@ -1,47 +1,48 @@
 
-.ceds-pagination {}
+.ceds-pagination {
 
-.ceds-pagination__summary {
-  @extend .govuk-grid-column-one-half;
-}
+  &__summary {
+    @extend .govuk-grid-column-one-half;
+  }
 
-.ceds-pagination__controls {
-  @extend .govuk-grid-column-one-half;
-  @extend .align-right;
-}
+  &__controls {
+    @extend .govuk-grid-column-one-half;
+    @extend .align-right;
+  }
 
-.ceds-pagination__items {
-  display: inline;
-}
+  &__items {
+    display: inline;
+  }
 
-// Can it be changed to class?
-.ceds-pagination__items .ceds-pagination__item {
-  display: inline;
-  margin: 0 5px;
-}
+  &__items &__item {
```

Comment:
I am not 100% sure if this construction in 100% - scss has some unexpected behaviour in terms of multiple & in directives

Reply:
It is working fine.